doc/nrf/app_build_system.rst

```diff
@@ -106,53 +106,53 @@ The following scenarios are possible:
-|                                 | (customizable by user, sample name by default).                                                 |                                                                          |
-|                                 | For example: :file:`127F-0141-01020003-light_switch.zigbee`.                                    |                                                                          |
-+---------------------------------+-------------------------------------------------------------------------------------------------+--------------------------------------------------------------------------+
++---------------------------------+-------------------------------------------------------------------------------------------------+-----------------------------------------------------------------------------------------------------------------+
+| File                            | Description                                                                                     | Programming scenario                                                                                            |
++=================================+=================================================================================================+=================================================================================================================+
+| :file:`zephyr.hex`              | Default full image.                                                                             | * Programming build targets with :ref:`NSPE <app_boards_spe_nspe>` or single-image.                             |
+|                                 | In a multi-image build, several :file:`zephyr.hex` are generated, one for each image.           | * Testing DFU procedure with nrfjprog (programming directly to device).                                         |
++---------------------------------+-------------------------------------------------------------------------------------------------+-----------------------------------------------------------------------------------------------------------------+
+| :file:`merged.hex`              | The result of merging all :file:`zephyr.hex` files for all images for a core                    | * Programming multi-core application.                                                                           |
+|                                 | in a multi-image build. Used by Nordic Semiconductor's build targets in single-core             | * Testing DFU procedure with nrfjprog (programming directly to device).                                         |
+|                                 | multi-image builds. In multi-core builds, several :file:`merged_<domain_name>.hex` fields       |                                                                                                                 |
```

Comment:
What about a more normalized name "image" instead of "domain". The first line already uses "all images" in the documentation

```suggestion
|                                 | multi-image builds. In multi-core builds, several :file:`merged_<domain_name>.hex` fields       |                                                                                                                 |
```

Reply:
Applied.